### ISSUE TITLE
Question about implementation

### ISSUE THREAD

**Issue by Sm00g15** [techniq/mui-downshift#65]
Do I need to use this, instead of regular Downshift, in order to get regular floating labels implemented properly?

**Comment by techniq** [comment]
Hmm.. I'm not 100% sure if I'm following your question, but this is the code that controls whether an input label should shrink or not.

https://github.com/techniq/mui-downshift/blob/72e89fcd841aee62e440ecfece5aa9c1e2b82fb4/src/Input.js#L45

**Comment by Sm00g15** [comment]
Yeah, so basically wondering if I need to use Mui downshift in order to be able to integrate floating labels into my downshift input or if that can be achieved with regular downshift rendering a standard mui TextInput

**Comment by techniq** [comment]
You should be able to accomplish this yourself using just `material-ui` and `downshift`.  This library just helps to marry the two together.  For example, I'm using mui's `FormControl`, `Input` and `InputLabel` instead of just [TextField](https://material-ui.com/demos/text-fields/) for more control (which is mentioned below on their [docs](https://material-ui.com/demos/text-fields/#components)).

So in short, you don't have to use my `mui-downshift` library, but it provides better support for more use cases you might run into if you implement the two yourself (see the [stories](https://techniq.github.io/mui-downshift/?selectedKind=Basic&selectedStory=defaults%20%28empty%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)).
